online-shop/src/css/ProductDetails.css

```diff
@@ -0,0 +1,29 @@
+.ProductDetailsGrid {
+    max-width: 50%;
+    padding-left: 5%;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.ProductDetailsItem {
+    color: azure;
+    text-align: left;
+    margin-top: 1em!important;
```

Comment:
Evita sa folosesti '!important'. Nu vad ce ar putea strica in situatia ta, insa in combinatie cu tag-uri de html da batai de cap.
https://uxengineer.com/css-specificity-avoid-important-css/